languages/swift/exercises/concept/enums/Sources/Enums/Enums.swift

```diff
@@ -0,0 +1 @@
+// TODO: define the 'LogLevel' enum
```

Comment:
As the enums methods are defined in the enum definition itself (I like this!), maybe we could add a little note about what methods we expect the student to add:

```suggestion
// TODO: define the 'LogLevel' enum and its `init` and `shortFormat` methods
```

or somewhat more generically:

```suggestion
// TODO: define the 'LogLevel' enum and its methods
```

That said, I'd also be totally fine with omitting this.